example-monorepos/blank/apps/next/next.config.js

```diff
@@ -13,12 +8,15 @@ const withTM = require('next-transpile-modules')([
   'dripsy',
   '@dripsy/core',
   'moti',
-  '@motify/core',
-  '@motify/components',
   'app',
 ])
 
-module.exports = withPlugins(
-  [withTM, withFonts, withImages, [withExpo, { projectRoot: __dirname }]],
-  nextConfig
-)
+const transform = withPlugins([withTM, withFonts, withImages, withExpo])
```

Comment:
I wonder if we should keep the `nextConfig` variable placed separately. This way we can get the JSDoc suggestions, and we can spread it onto here.

Also, is the `projectRoot` not needed from Expo?